lib/3scale/backend/errors.rb

```diff
@@ -297,5 +300,23 @@ def initialize
         super 'End-users are no longer supported, do not specify the user_id parameter'.freeze
       end
     end
+
+    class PipelineSharedBetweenFibers < Error
```

Comment:
While I agree on the change, I'm not sure it's appropriate because all other errors in this file lack the `Error` suffix as well, except `AuthenticationError`. But if you insist I'll rename it.

Same for the other suggestion about `BrokenPipeline`

Reply:
huh, all good to be but another time I guess